tests/testthat/test-cache.R

```diff
@@ -60,8 +60,7 @@ test_that("load_cache loads the saved file in a new empty environment", {
   save_cache(cache = e1, file = f1, path = d1)
   e2 <- load_cache(file = f1, path = d1)
 
-  # TODO(michaelchirico): use plain expect_equal after waldo#133 makes it into a CRAN release
-  expect_equal(as.list(e2, all.names = TRUE), as.list(e1, all.names = TRUE))
+  expect_equal(e2, e1)
```

Comment:
don't we need to add a version dependency to DESCRIPTION as well?

Reply:
We can bump the needed `{testthat}` version only for this particular test.

Reply:
let's wait until the patch release hits CRAN for this

Reply:
Yepp. I am not merging anything, even if the PRs are approved (e.g. #1522).